app/components/views/SettingsPage/SettingsPage.jsx

```diff
@@ -17,19 +17,7 @@ const closeWalletModalContent = (walletName) => (
   />
 );
 
-const closeWalletWithAutobuyerModal = (walletName) => (
-  <T
-    id="settings.closeWalletModalWithAutobuyerModal"
```

Comment:
I dont think this should have been removed. I am trying to close a wallet with the ticketbuyer running, and I see the action being logged
```
 action     {type: "SHOW_AUTOBUYER_RUNNING_MODAL"}
```
but nothing happens at all. I am not getting any popup modal, and decrediton does not close.